script.js

```diff
@@ -0,0 +1,128 @@
+let myLibrary = [];
+
+function Book(title, author, pages, read){
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.read = read;
+}
+
+let newBookBtn = document.querySelector('.newBook-btn');
+newBookBtn.addEventListener('click', function(){
+     let newBookForm = document.querySelector('.newBook-form')
+     newBookForm.style.display = 'block';
+})
+
+function addBookToLibrary(){
+   let title = document.querySelector('#title').value
+   let author = document.querySelector('#author').value
+   let pages = document.querySelector('#pages').value
+   let reads = document.querySelector('#reads').checked
+   let newBook = new Book(title, author, pages, reads);
+   myLibrary.push(newBook);
+   render();
+
+}
+
+function createBook(item){
+    const bookGrid = document.querySelector('#booksGrid');
+    const bookDiv = document.createElement('div');
+    const titleDiv = document.createElement('div');
+    const authDiv = document.createElement('div');
+    const pageDiv = document.createElement('div');
+    const removeBtn = document.createElement('button');
+    const readBtn = document.createElement('button');
+
+    bookDiv.classList.add('book');
+    bookDiv.setAttribute('id', myLibrary.indexOf(item));
+
+    titleDiv.textContent = item.title;
+    titleDiv.classList.add('title');
+    bookDiv.appendChild(titleDiv);
+
+    authDiv.textContent = item.author;
+    authDiv.classList.add('author');
+    bookDiv.appendChild(authDiv);
+
+    pageDiv.textContent = item.pages;
+    pageDiv.classList.add('pages');
+    bookDiv.appendChild(pageDiv);
+
+    readBtn.classList.add('readBtn'); 
+    
+    readBtn.addEventListener('click', () => { 
+        item.read = !item.read;
+        setData();
+        render();
+    }); 
+    bookDiv.appendChild(readBtn);
+
+    if(item.read===false) {

```

Comment:
You can make the condition for this `if` block better

Reply:
ok sir.